src/ChemicalDiffusion/ChemicalDiffusion.jl

```diff
@@ -0,0 +1,349 @@
+module ChemicalDiffusion
+
+using GeoParams
+using Unitful
+
+using Parameters, LaTeXStrings, Unitful, MuladdMacro
```

Comment:
```suggestion
using Parameters, LaTeXStrings, Unitful
```
`MuladdMacro` is needed only if you use `@muladd`